script/domVisitor.ts

```diff
@@ -12,6 +13,15 @@ import createOSWebLoader from '../src/gateways/createOSWebLoader';
 import { findBooks } from './utils/bookUtils';
 import progressBar from './utils/progressBar';
 
+// Blocking GTM prevents most analytics scripts from loading
+const blockedRequests = [
+  /^https?:\/\/(?:www\.)?googletagmanager\.com/,
+  /^https?:\/\/js\.pulseinsights\.com/,
+];
+
+// Most pages perform less than 10 requests when GTM is blocked
+const cacheMaxSize = 50;
```

Comment:
Is this 50 globally, and is that enough for an entire book to not see misses?

Reply:
It's 50 requests in the LRU cache. From my testing this seems enough to get a pretty good hit rate, mainly because blocking GTM greatly reduces the number of requests (to about 10 per page).